image_batch/convert_png_to_jpg

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+for source_file in *.png; do
+  common_name=$(basename $source_file '.png')
```

Comment:
In order to support file containing spaces you should change this line to `common_name=$(basename "$source_file" '.png')` and the line 9 to `convert "$source_file" "$dest_file`

Reply:
oooo spaces. Gross but realistic. That's a good catch.